Wikipedia/Code/QuadKey.swift

```diff
@@ -96,18 +98,30 @@ public extension QuadKeyDegrees {
     public static func max(_ a: QuadKeyDegrees, _ b: QuadKeyDegrees) -> QuadKeyDegrees {
         return a > b ? a : b
     }
+    
+    public static func min(_ a: QuadKeyDegrees, _ b: QuadKeyDegrees) -> QuadKeyDegrees {
+        return a < b ? a : b
```

Comment:
Any possible equality issues? Guessing not, but looks like both methods always return `b` in that case.